app/services/CheckoutService.scala

```diff
@@ -39,12 +40,26 @@ class CheckoutService(identityService: IdentityService, salesforceService: Sales
         identityService.registerGuest(subscriptionData.personalData)
       }
 
+    userOrElseRegisterGuest.onComplete(t => Logger.info("userOrElseRegisterGuest result "+t))
+
+
     //TODO when implementing test-users this requires updating to supply data to correct location
     for {
       userData <- userOrElseRegisterGuest
-      memberId <- salesforceService.createOrUpdateUser(subscriptionData.personalData, userData.id)
-      subscribeResult <- zuoraService.createSubscription(memberId, subscriptionData)
+      memberId <- {
+        val mi = salesforceService.createOrUpdateUser(subscriptionData.personalData, userData.id)
+        mi.onComplete(t => Logger.info("salesforce result "+t))
+        mi
+      }
+      subscribeResult <- {
+        val sr = zuoraService.createSubscription(memberId, subscriptionData)
+        sr.onComplete(t => Logger.info("zuora result " + t))
+        sr
```

Comment:
are these logging statement still needed? if so this pattern can be simplified by using a map that does logging and returns the original object.